Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/CollectionEnums/ObjectOrientationSurfaceType.cs

```diff
@@ -11,6 +11,7 @@ public enum ObjectOrientationSurfaceType
         Cylinder,
         Plane,
         Sphere,
-        Scatter,
+        Radial,
+        Scatter
```

Comment:
Was this reordering intentional? I generally try to avoid renumbering enums when I add a new one, in part because it breaks existing serialized data. But more generally a philosophy of "if it's not broken, don't touch it". :)

Reply:
Works for me,  I put it at the bottom of the list for preference.